README.md

```diff
@@ -12,28 +12,62 @@ After you create a new repository using this repo as template, please follow the
 4. Adjust pre-commit configuration file, `.pre-commit-config.yaml` to enable/disable the hooks you need. For more information about pre-commit, please refer to [pre-commit](https://pre-commit.com/).
 
+
 5. Having code coverage, `codecov.yaml` is **strongly recommended**, please refer to [Code coverage](https://coverage.readthedocs.io/en/coverage-5.5/) for more information.
 
+
 6. Adjust the demo Github action yaml files for CI/CD. For more information about Github action, please refer to [Github action](https://docs.github.com/en/actions).
 
+    6.1 Specify package name at: https://github.com/neutrons/python_project_template/blob/df49fa434591c073fe88f50d419b14c789fb1e17/.github/workflows/package.yml#L34
+
+    6.2 Specify package name at: https://github.com/neutrons/python_project_template/blob/df49fa434591c073fe88f50d419b14c789fb1e17/.github/workflows/package.yml#L46
+
+
 7. Adjust the conda recipe, `conda-recipe/meta.yaml` to provide the meta information for the conda package. For more information about conda recipe, please refer to [Conda build](https://docs.conda.io/projects/conda-build/en/latest/).
 
+    7.1 Specify package name at: https://github.com/neutrons/python_project_template/blob/df49fa434591c073fe88f50d419b14c789fb1e17/conda.recipe/meta.yaml#L15
+
+    7.2 Update license family, if necessary: https://github.com/neutrons/python_project_template/blob/df49fa434591c073fe88f50d419b14c789fb1e17/conda.recipe/meta.yaml#L42
+
+
 8. Adjust `pyproject.toml` to match your project. For more information about `pyproject.toml`, please refer to [pyproject.toml](https://www.python.org/dev/peps/pep-0518/).
 
-    8.1 We strongly recommended using a single `pyproject.toml` file to manage all the project metadata, including the project name, version, author, license, etc.
-    8.2 Python is moving away from `setup.cfg`/`setup.py`, and we would like to follow the trend for our new projects.
+    8.1 Specify package name at: https://github.com/neutrons/python_project_template/blob/df49fa434591c073fe88f50d419b14c789fb1e17/pyproject.toml#L2
+
+    8.2 Specify package description at: https://github.com/neutrons/python_project_template/blob/df49fa434591c073fe88f50d419b14c789fb1e17/pyproject.toml#L3
+
+    8.3 Specify package name at: https://github.com/neutrons/python_project_template/blob/df49fa434591c073fe88f50d419b14c789fb1e17/pyproject.toml#L40
+
+    8.4 We strongly recommended using a single `pyproject.toml` file to manage all the project metadata, including the project name, version, author, license, etc.
+
+    8.5 Python is moving away from `setup.cfg`/`setup.py`, and we would like to follow the trend for our new projects.
+
+
+9. Specify package name at  https://github.com/neutrons/python_project_template/tree/df49fa434591c073fe88f50d419b14c789fb1e17/src/packagenamepy
+
+
+10. Adjust 'src/packagenamepy/__main__.py: https://github.com/neutrons/python_project_template/blob/df49fa434591c073fe88f50d419b14c789fb1e17/src/packagenamepy/__main__.py
+
```

Comment:
please see my comments on #5 about using permanent link in readme.

Reply:
Working backwards on that, up next